src/server/models/db/migrations/20230427161921_add_view_ListsForDatabase/migration.sql

```diff
@@ -0,0 +1,20 @@
+-- This is a migration to help get the correct data for the relevant rows in on the list management dashboard page
+create view "ListsForDashboard" as (
+  select
+    l.id "listId",
+    l.type "type",
+    c.name "country",
+    l."nextAnnualReviewStartDate" "nextAnnualReviewStartDate",
+    l."lastAnnualReviewStartDate" "lastAnnualReviewStartDate",
+    l."jsonData" "jsonData",
+    (case when coalesce(l."nextAnnualReviewStartDate", fp."firstPublished")  + interval '18 months' < current_date then true else false end) as "isOverdue",
```

Comment:
I've tried it without the case and sometimes isOverdue comes up blank. I think it's because of the outer join the coalesce function could have two nulls
<img width="1489" alt="Screenshot 2023-05-10 at 11 20 49" src="https://github.com/UKForeignOffice/lists/assets/1377253/fa16790c-9f88-42f6-977c-ae111e453cba">